src/navigation/BottomTabNavigator.tsx

```diff
@@ -35,6 +35,12 @@ const BottomTabNavigator = (): JSX.Element | null => {
 
   const iconSize = hp('3.5%')
 
+  const [repetitionBadgeNumber, setRepetitionBadgeNumber] = useState<undefined | number>(undefined)
+
+  getNumberOfWordNodeCardsNeedingRepetitionWithUpperBound()
+    .then(result => setRepetitionBadgeNumber(result > 0 ? result : undefined))
+    .catch(() => setRepetitionBadgeNumber(undefined))
```

Comment:
I think we should use our `useLoadAsync` hook for this for lifecycle stuff and correct handling of errors and such :) 

```suggestion
 const { data: repetitionBadgeNumber } = useLoadAsync(
    getNumberOfWordNodeCardsNeedingRepetitionWithUpperBound,
    undefined
  )
```
This would also make updating the number (e.g. after doing repetition exercise) later as we can just call `refresh`.